layouts/README.md

```diff
@@ -0,0 +1,7 @@
+# LAYOUTS
```

Comment:
Let's delete this one for now.

Reply:
even with this file not present, i still think it will be solved as a default layout by nuxt, no ?

i will remove this anyways